tests/integration/conftest.py

```diff
@@ -251,6 +252,10 @@ def docker_compose_file(pytestconfig, netbox_docker_repo_dirpaths):
                         "netboxcommunity/netbox:v%s" % netbox_version
                     )
 
+                if service_name == "netbox":
+                    # ensure the netbox container listens on a random port
+                    new_services[new_service_name]["ports"] = ["8080"]
+
```

Comment:
:bow: this was foiling my attempts to get this going a few weeks ago :laughing:

Reply:
Yeah, they apparently removed all default `port` listener statements from the default config so that people could have more control. Makes sense but I wish the project was a little more backwards compatible :)